### ISSUE TITLE
Complie Errors when run the Allwmake script of branch multiDimAMR-v2012

### ISSUE THREAD

**Issue by kaiserpeng** [HenningScheufler/multiDimAMR#4]
Dear @HenningScheufler , I am sure of that I had tried to compile multiDimAMR-v2012 into the newest Openfoam-dev(v2106) for several times,  but still the error happended  when running Allwmake script of multiDimAMR-v2012.
![AMRLB-compiler error](https://user-images.githubusercontent.com/26056456/123646141-86c9e780-d859-11eb-9b27-72e375bb6f97.png)
the complete compiled log  was attached
[log.zip](https://github.com/HenningScheufler/multiDimAMR/files/6726757/log.zip)

**Comment by HenningScheufler** [comment]
It should work. 

Try:

```
wclean all 
wclean dep
./Allwmake
```

Could you also verify that you are on the correct branch with 

git log and post the commit

**Comment by kaiserpeng** [comment]
Dear @HenningScheufler  
Thanks for your reply， I have tried your advice and the problem still not been solved.
But I have found the reason according to the error log. The error was not only found for AMRLB library, see the link below.
Maybe I guess it caused by the gcc version. So I changed the gcc version from 9.1.0 to 7.3.0.
Then , it works and the tutorial case was run successfully.
Thanks very much anyhow
[https://www.cfd-online.com/Forums/openfoam-programming-development/200083-undefined-reference-foam-uopstream-writequoted.html]()
